hasktorch/src/Torch/Typed/NN/Transformer.hs

```diff
@@ -581,10 +581,16 @@ instance
     KnownDType dtype,
     KnownDevice device
   ) =>
-  HasForward (TransformerLM numAttnLayers numHeads ffnDim paddingIdx numEmbeds embedDim dtype device) (Tensor device 'D.Int64 '[batchSize, seqLen]) (Tensor device dtype '[batchSize, seqLen, numEmbeds])
+  HasForward
+    (TransformerLM numAttnLayers numHeads ffnDim paddingIdx numEmbeds embedDim dtype device)
+    (Tensor device 'D.Int64 '[batchSize, seqLen])
   where
+  type
+    Output
+      (TransformerLM numAttnLayers numHeads ffnDim paddingIdx numEmbeds embedDim dtype device)
+      (Tensor device 'D.Int64 '[batchSize, seqLen]) =
+      Tensor device dtype '[batchSize, seqLen, numEmbeds]
   forward model input = unsafePerformIO $ transformerLM model False input
```

Comment:
same here. I hope we can get a dropout soon that takes a generator as argument. otherwise, we can't train these models with dropout.

Reply:
this still isn't resolved :/

Reply:
Yeah, too bad. When I find some time I'm hoping to try to reproduce the test failure in Junji's PR and see if I can make any progress..